service/status-report/main.go

```diff
@@ -215,6 +222,25 @@ func setClusterInformation(ctx context.Context, cl client.Client, status interfa
 
 }
 
+func setStorageQuotaUtilizationRatio(ctx context.Context, cl client.Client, status interfaces.StorageClientStatus) {
+	clusterResourceQuota := &quotav1.ClusterResourceQuota{}
+	clusterResourceQuota.Name = status.GetClientName()
+
+	if err := cl.Get(ctx, client.ObjectKeyFromObject(clusterResourceQuota), clusterResourceQuota); err != nil {
+		// No need to check for NotFound because unlimited quota client will not have CRQ resource
+		if !errors.IsNotFound(err) {
+			klog.Warningf("Failed to get clusterResourceQuota %q: %v", clusterResourceQuota.Name, err)
+		}
+	}
+	if clusterResourceQuota.Status.Total.Hard != nil {
+		var total = clusterResourceQuota.Status.Total.Hard["requests.storage"]
```

Comment:
total will never be zero, isn't it?

Reply:
it can be zero incase of unlimited quota.

Reply:
I mean in this block, if it's zero it might panic w/ dividebyzero isn't it?

Reply:
ohh yeah, I got your point. will handle that condition